src/libs/wds-status.test.ts

```diff
@@ -126,7 +126,6 @@ describe('useWdsStatus', () => {
       },
       appName: 'wds-6601fdbb-4b53-41da-87b2-81385f4a760e',
       appType: 'WDS',
-      // @ts-expect-error Leo app types do not reflect actual API responses.
```

Comment:
hmm.. why is this needed?

Reply:
VScode was telling me that they are unused, but I can put it back!

Reply:
This comment was originally here because I added a realistic mock response in this test in anticipation of IA-4439 getting resolved. It's good that this comment is no longer necessary.